<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CadTomografias</class>
 <widget class="QMainWindow" name="CadTomografias">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>806</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Id</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="edtId"/>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Paciente</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLineEdit" name="edtPaciente">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnLimparPaciente">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Limpar</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnPesqPaciente">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Pesquisar</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Aluno</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLineEdit" name="edtAluno">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnLimparAluno">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Limpar</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnPesqAluno">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Pesquisar</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Valor</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="edtValor">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="chkEntregue">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Entregue</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Data de Entrega</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDateEdit" name="edtEntrega">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Motivo</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="edtMotivo">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Tipo</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="cboTipo">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <item>
         <property name="text">
          <string>Maxila</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Mandibula</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>ATM</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Segmento</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Elemento</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Especialização</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="cboEspecializacao">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <item>
         <property name="text">
          <string>Nanhuma</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Ramos Mandibulares</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Oclusao</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Abertura</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>Região</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="edtRegiao">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>Elemento</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="edtElemento">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <widget class="QLabel" name="label_13">
            <property name="text">
             <string>Resolução</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="cboResolucao">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <item>
             <property name="text">
              <string>Baixa</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Média</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Alta</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QLabel" name="label_12">
            <property name="text">
             <string>Alvo</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="cboAlvo">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <item>
             <property name="text">
              <string>Alvo</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Dente</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QLabel" name="label_11">
            <property name="text">
             <string>Proporção</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="cboProporcao">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <item>
             <property name="text">
              <string>5x5</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>6x8</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>8x8</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>8x15</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>13x15</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="btnParametrizacao">
        <property name="text">
         <string>Parametrização</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QPushButton" name="btnNovo">
        <property name="text">
         <string>&amp;Novo</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnSalvar">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>&amp;Salvar</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnPesquisar">
        <property name="text">
         <string>&amp;Pesquisar</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnRemover">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>&amp;Remover</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnSair">
        <property name="text">
         <string>Sai&amp;r</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>806</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
